modules/dnn/src/tensorflow/tf_importer.cpp

```diff
@@ -1290,29 +1335,27 @@ void TFImporter::populateNet(Net dstNet)
                 connect(layer_id, dstNet, inp, id, ii - from);
             }
         }
-        else if (type == "MaxPool")
+        else if (type == "MaxPool" || type == "MaxPool3D")
         {
             layerParams.set("pool", "max");
 
             setKSize(layerParams, layer);
             setStrides(layerParams, layer);
             setPadding(layerParams, layer);
-
```

Comment:
I don't mind but is it critical to remove this empty line? :D

Reply:
done